docker/install_docker.sh

```diff
@@ -156,45 +155,52 @@ if [ "$DOCKER_COMPOSE_CHECK" -eq 0 ]; then
 	echo "Docker-Compose appears to already be installed. Skipping."
 else
 	### Install Docker-Compose ###
-	# https://docs.docker.com/compose/install/#install-compose
-	DOCKER_COMPOSE_VERSION="1.29.2"
-
-	echo "Installing Docker-Compose v${DOCKER_COMPOSE_VERSION}..."
-
-	# Check if the latest version of pip is supported by the version of python installed
-	# In particular, Ubuntu 16's version of python (v3.5) does not support the latest verison of pip
-	MIN_PYTHON_VERSION_MAJOR=3
-	MIN_PYTHON_VERSION_MINOR=6
-	PYTHON_VERSION_TEST="
-import sys
-if  (sys.version_info.major > $MIN_PYTHON_VERSION_MAJOR or 
-	(sys.version_info.major == $MIN_PYTHON_VERSION_MAJOR and sys.version_info.minor >= $MIN_PYTHON_VERSION_MINOR)): 
-	sys.exit(0)
-sys.exit(1)
-"
-	if python3 -c "$PYTHON_VERSION_TEST"; then 
-		# prefer to install with pip3 if possible since github doesn't have aarch64 binary releases for docker-compose
+	# https://docs.docker.com/compose/install/linux/	
+
+	echo "Installing Docker Compose..."
+
+	# Uninstall docker-compose@v1 installed via pip
+	PIP3_CMD="python3 -m pip"
+	if [ -n "$SUDO" ]; then
+		PIP3_CMD="$SUDO -H $PIP3_CMD"
+	fi
 
-		# pip3 recommends -H when running with sudo to prevent creating root owned files in the user's home dir
-		PIP3_CMD="python3 -m pip"
-		if [ -n "$SUDO" ]; then
-			PIP3_CMD="$SUDO -H $PIP3_CMD"
+	# Check if pip is callable
+	if "$PIP3_CMD" --version > /dev/null 2>&1 ; then 
+		# Check if docker-compose is installed via pip
+		if [ "$PIP3_CMD" list | grep -s "docker-compose" ]; then
+			# Attempt to uninstall docker-compose
+			if [ "$PIP3_CMD" uninstall docker-compose ]; then
+				echo "Uninstalled docker-compose@v1"
+			fi
 		fi
-		$PIP3_CMD install --upgrade pip
-		$PIP3_CMD install --no-warn-script-location docker-compose==${DOCKER_COMPOSE_VERSION}
-	elif [ "$(uname -m)" = "x86_64" ]; then
-		# if we are on x86, download docker-compose from Github
-		$SUDO_E curl --silent -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
-		$SUDO chmod +x /usr/local/bin/docker-compose
-	else 
-		fail 'docker-compose could not be automatically installed on this system. Please install it manually and re-run the script.'
 	fi
 
-	# Some OS don't insert /usr/local/bin into the PATH when running SUDO (CentOS)
-	# Provide a symlink in /usr/bin in order to get around this issue.
+	# Update package lists again because docker adds the docker-compose-plugin repo
+	# Install docker-compose-plugin
+	if [ -s /etc/redhat-release ] && grep -iq 'release 7\|release 8' /etc/redhat-release ; then
+		# TODO: add silencing flags
+		$SUDO yum update
```

Comment:
There is a TODO here already marking this change, but I'm leaving a note here to track it in the PR review.

We need to pass `-q -y` here.